Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -345,6 +344,23 @@ GDCMImageIO::Read(void * pointer)
     len = len * outputpt.GetPixelSize() / pixeltype.GetPixelSize();
   }
 
+  // Y'CbCr to RGB
+  // GDCM 3.0.3
+  const bool ybr =
+    m_NumberOfComponents == 3 &&
+    (pi == gdcm::PhotometricInterpretation::YBR_FULL ||
+     pi == gdcm::PhotometricInterpretation::YBR_FULL_422) &&
+    (pixeltype == gdcm::PixelFormat::UINT8 ||
+     pixeltype == gdcm::PixelFormat::INT8);
```

Comment:
`pixeltype == gdcm::PixelFormat::INT8` are you sure about that ?

Reply:
Yes, highly likely it will be always UINT8, but there is no harm to add INT8 here.